src/AppInstallerCLICore/Workflows/MSStoreInstallerHandler.cpp

```diff
@@ -115,6 +115,12 @@ namespace AppInstaller::CLI::Workflow
             }
             else if (result.Status() == GetEntitlementStatus::ServerError)
             {
+                bool bypassStorePolicy = WI_IsFlagSet(context.GetFlags(), Execution::ContextFlag::BypassStorePolicy);

```

Comment:
This is not Store client policy check, please revert

Reply:
Our previous conversation with the api owner says these apis do not check for the policy. I'll need to try locally to confirm